deploy/vagrant/create

```diff
@@ -70,6 +70,13 @@ if [[ "$PROVIDER" == "aws" ]]; then
   fi
 fi
 
+# make sure gce.yml exists
+if [[ "$PROVIDER" == “google” ]]; then
```

Comment:
as per @ifcharming's comment, we try to enforce the above Google shell style guide for any new code even if there is old code that does not adhere to it